backend/compact-connect/lambdas/staff-users/tests/function/test_handlers/test_patch_user.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+from moto import mock_aws
+
+from tests.function import TstFunction
+
+
+@mock_aws
+class TestPatchUser(TstFunction):
+    def test_patch_user(self):
+        self._load_user_data()
+
+        from handlers.users import patch_user
+
+        with open('tests/resources/api-event.json', 'r') as f:
+            event = json.load(f)
+
+        # The user has admin permission for all of aslp
```

Comment:
Do we (and should we) test ability of non-admin users to patch other user data?

Reply:
You mean like an admin trying to patch user attributes instead of permissions? I suppose we could, though it would take some serious refactoring of the app to the extent that it would be malicious for something like that to become possible since:
1) the API schema does not allow for user attributes to even be provided in the request body for this endpoint;
2) this endpoint doesn't even look for `attributes` if they got past schema validation, so it would be ignored; and
3) the `UserClient` method for updating attributes is separate from the one that is used to update permissions and is not called here anyway.

Reply:
Ah sorry, I mean a non-admin (e.g., a provider) trying to hit this endpoint.

Reply:
Oh, right. Yes, yes we should.